main.py

```diff
@@ -1,16 +1,34 @@
-# This is a sample Python script.
+from flask import Flask, render_template, redirect, url_for
+from flask_wtf.csrf import CSRFProtect
+from hashlib import sha256
 
-# Press Shift+F10 to execute it or replace it with your code.
-# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+from data_base import db, User
+from form import LoginFrom
 
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///mydatabase.db'
 
-def print_hi(name):
-    # Use a breakpoint in the code line below to debug your script.
-    print(f'Hi, {name}')  # Press Ctrl+F8 to toggle the breakpoint.
+app.config['SECRET_KEY'] = 'mysecretkey'
+csrf = CSRFProtect(app)
 
 
-# Press the green button in the gutter to run the script.
-if __name__ == '__main__':
-    print_hi('PyCharm')
+@app.cli.command("init-db")
+def init_db():
+    db.create_all()
+
+@app.route("/")
+def login():
+    form = LoginFrom()
+    if form.validate_on_submit():
+        new_user = User(first_name=form.first_name.data,
+                        second_name=form.second_name.data,
+                        email=form.email.data,
+                        paswword=sha256(form.password.data.encond(enconding="UTF-8")).hexdigest())
+        db.session.add(new_user)
+        db.session.commit()
+        return redirect(url_for("login"))
```

Comment:
А где у вас обрабатываются сообщения на случай, если форма не прошла валидацию или на случай, если всё таки данные были записаны в базу данных? Например, можно использовать `flash` функцию
```suggestion
        db.session.commit()
        flash('Регистрация прошла успешно')
        return redirect(url_for("login"))
     else:
        for field in form:
            for error in field.errors:
                flash(error)
```